integrate-kobiton-travisci.md

```diff
@@ -0,0 +1,246 @@
+
+  **2.1 Get username and API key**   
+  Go to https://portal.kobiton.com and login to your Kobiton account.
+  - Get username
+    * Select *user icon* -> **"Profile"** (you might find it in the top right corner of the navigation bar)
+  
+    ![](assets/2_kobiton_username.jpg)
+  - Get API key
+    * Select *user icon* -> **"Settings"**
+  
+    ![](assets/2_kobiton_apikey.jpg)
+
+  **2.2 Get desired capabilities**  
+    The desired capabilities need to be added to the automation test script in order tests to be executed on the Kobiton device.
+
+  Go to https://portal.kobiton.com and login to your Kobiton account.
+  * In the top navigation bar, select **"Devices"**.
+    ![](assets/1_device_bar.jpg)
+  * Hover over any device you want to test with and click on the Automation settings button (the gear symbol)   
+  
+    ![](assets/2_get_device.jpg)
+  * You can choose your preferred language, app type. In this guideline, we will choose **"NodeJS"** for *Language* and **"Hybrid/Native from Apps"** for *App type*. A block of code will be generated on the right column. Write down the code as we will be using it later.
+  
+    ![](assets/2_kobiton_device.jpg)
+
+
+## 3. Adding required enviroment variables to Travis CI
+
+Because variables such as `API Key` and `Hockey App Token` are sensitive, we must secure these elements with Travis CI to use in our script.
+1. Go to your Travis CI project.
+2. Click on **"More options"** and select **"Settings"**
+   
+![](assets/travis_settings.png)
+
+3. In the input dialog, type in your enviroment variables we need.
+ 
+To secure sensitive variables, please make sure to **uncheck** **"Display value in build log"**
+
+Click **"Add"** to confirm.
+
+![](assets/travis_add_env.png)
+
+Here are the variables we need. For the sake of this guideline, please set the name for these variables like below
+
+`APP_ID`: Your app ID
+
+`FILE_FORMAT`: Your app file extension. `apk` for Android, `ipa` for iOS.
+
+`HOCKEYAPP_TOKEN`: Hockey App API token
+
+`KOBITON_API_KEY`: Your Kobiton API Key
+
+`KOBITON_USERNAME`: Your Kobiton username
+
+![](assets/travis_env.png)
+
+>For other methods to secure your enviromental variables, visit [here.](https://docs.travis-ci.com/user/environment-variables/)
+
+
+## 4. Configuring to test latest app version on Kobiton devices
+
+This section will guide you to fetch the latest version of your app on Hockey App and execute automation test on Kobiton devices with it.
+
+The main steps of this section:
+1. Use Hockey App API to get the latest app version and its download URL.
+2. Setting desired capabilities
+3. Adding test script execution command
+
+### 4.1. Fetching the latest app version and app download URL using Hockey App API
+
+To make things simplier, we have provided a bash script that will handle the first step as well as execute your automation test script.
```

Comment:
**simpler** or **more simple**
I think you should clearly explain the `first step`

Reply:
What if they want to separate it into two steps ?